Content.Shared/Traits/Assorted/AccentlessSystem.cs

```diff
@@ -23,4 +26,12 @@ private void RemoveAccents(EntityUid uid, AccentlessComponent component, Compone
             RemComp(uid, accentComponent.GetType());
         }
     }
+	
+	private void OnCloneFinished(EntityUid uid, AccentlessComponent component, CloneFinishedEvent args)
+    {
+		
+        RemoveAccents(uid, component, new ComponentStartup());
+    
+	}
```

Comment:
Ну что это за огород
```suggestion
	private void OnCloneFinished(EntityUid uid, AccentlessComponent component, CloneFinishedEvent args)
    {
        RemoveAccents(uid, component, new ComponentStartup());
	}
```